.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -22,8 +22,24 @@
 //  http://groovy-lang.org/style-guide.html
 class common_job_properties {
 
+  static void setBeamSCM(def context, String repositoryName) {
```

Comment:
<!--new_thread; commit:13434ff7764e80f3ff96f6120b9100f537df0f36; resolved:0-->
This combination of naming and the variable for "beam" seems strange to me.